pkg/event/target/webhook.go

```diff
@@ -79,12 +79,11 @@ func (target *WebhookTarget) Send(eventData event.Event) error {
 	// FIXME: log returned error. ignore time being.
 	_ = resp.Body.Close()
 
-	switch resp.StatusCode {
-	case http.StatusOK, http.StatusAccepted, http.StatusContinue:
-		return nil
-	default:
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
```

Comment:
Instead of numbers, is it better to use constants like this ?
`if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {`

Reply:
No. It's wrong. You need to check rfc in previous comments.

Reply:
I am assuming that you mean that the rfc says `2xx` means success and it might be easier to read, if we have numbers than constants here.